spec/student_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe Student do
+  before :each do
+    @student = Student.new 'Title', 'Author'
```

Comment:
- [ ] You also need to write tests for positive and negative scenarios. For example, you have parental permission. Or can use services. So you can decide which of them you choose. But you must have written tests for both scenarios. Kindly add them.